.gitignore

```diff
@@ -7,6 +7,7 @@ dist/
 htmlcov/
 coverage.xml
 .cache/
+*.pyc
```

Comment:
You should not have any `*.pyc` files apart from thoses already ignored by the `__pycache__` ignore, as we're only supporting python3(.5)